src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -71,18 +73,19 @@ public synchronized void addEntry(ReadOnlyEntry entry) {
     }
 
     @Override
-    public void updateEntry(ReadOnlyEntry target, ReadOnlyEntry editedEntry)
-            throws EntryNotFoundException {
+    public void updateEntry(ReadOnlyEntry target, ReadOnlyEntry editedEntry) throws EntryNotFoundException {
         requireAllNonNull(target, editedEntry);
 
         entryBook.updateEntry(target, editedEntry);
         indicateEntryBookChanged();
     }
 
-    //=========== Filtered Entry List Accessors =============================================================
+    // =========== Filtered Entry List Accessors
+    // =============================================================
```

Comment:
@kevinLamKB 
Okay, lol Eclipse auto-format messed up the comments quite a bit. The only change I did is at lines 103-106.